src/test.cpp

```diff
@@ -109,14 +109,15 @@ void TestExhaustive(uint32_t bits, size_t capacity) {
                 // Compare
                 CHECK(serialized == serialized_rebuild);
                 // Count it
-                if (elements_0.size() <= capacity) ++counts[elements_0.size()];
+                if (impl == 0 && elements_0.size() <= capacity) ++counts[elements_0.size()];
```

Comment:
So this line changes nothing, it just saves us from rewriting something again with the same value?

Reply:
Without out, `counts[...]` will be tripled if there are 3 implementations.